src/arrayMethodSort.js

```diff
@@ -1,11 +1,30 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const compareFn =
+      compareFunction ||
+      function (a, b) {
+        if (a.toString() > b.toString()) {
+          return 1;
+        }
+
+        if (a.toString() < b.toString()) {
+          return -1;
+        }
+
+        return 0;
+      };
+
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let j = 0; j < this.length - 1 - i; j++) {
+        if (compareFn(this[j], this[j + 1]) > 0) {
```

Comment:
Using array destructuring for swapping elements is a modern and clean approach, well done. Just make sure to remember that this method may have slightly more overhead than using a temporary variable due to the creation of temporary arrays.